pages/dmp_supporting_info/support_00_dmp_requirements.md

```diff
@@ -48,15 +52,22 @@ DMP requirements are commonly part of Open Science policies or guidelines for re
 > * Research data must be made accessible for reuse
 > * Reserach data should be made accessible at an early stage [latest at publication]
 > * Research data must be accompanied by standardized metadata
-> * Research dataprovided with a license for access, reuse, and redistribution
+> * Research data provided with a license for access, reuse, and redistribution
 > * Research data should be made accessible at the lowest possible cost [preferable at no charge]
-> * The management of research data must be described in a data management plan [DMP must be delivered in connection with the revised grant application, final version together with the final report]
+> * The management of research data must be described in a data management plan
+>   * Projects must submit the first version of the plan when revising the grant application
+>   * An updated version is delivered together with the project's final report
+>   * The Research Council of Norway does not assess the content of submitted plans. It is the responsibility of the Project Owner to approve that the plan is in line with the institution's requirements and guidelines before it is submitted
```

Comment:
Dette står på nettsiden men ikke i policyen fra 2017 som vises til?

Reply:
Jeg skjønner at det blir feil, når vi har en overskrift som henviser til policy-dokumentet men der legger inn info som er hentet fra andre steder. Det jeg tenker er at vi bør gjøre dette mest mulig brukervennlig. Og da blir det vel mest brukervennlig om vi syr sammen info fra ulike steder(?) Men vi må være ryddige på hvor sitatene hentes fra. Kunne vi gjort om og kun hatt én overskrift for NFR-info, men så bruke noter som viser hvor hvert sitat er hentet fra?